docs/app/pages/css/css-sections/icons/ux-icons/icon-preview/icon-snippet/icon-snippet.component.less

```diff
@@ -0,0 +1,20 @@
+:host {
+   display: inline-flex;
+    padding: 10px 0;
+    width: 100%;
+
+    uxd-snippet {
+      display:flex;
+        flex: 1;
+    }
+
+    button {
```

Comment:
Might be good to use classes here instead of element selectors in case we ever add another button in to this component then it may get these styles unexpectedly

Reply:
Added classes for elements